### ISSUE TITLE
Using this async

### ISSUE THREAD

**Issue by vijmen** [podo-os/dlib-face-recognition#33]
Would you be able to provide an async example for face recognition from video streams using something like tokio?

**Comment by HoKim98** [comment]
Do you mean a practical example on tokio? I have few time to focus on so it would be slow. Would you affordable so?

**Comment by vijmen** [comment]
Yes, a practical example as the async version seems to be quite complicated. Regards On 27 May 2024, at 3:29 PM, Ho Kim ***@***.***> wrote:﻿
Do you mean a practical example on tokio? I have few time to focus on so it would be slow. Would you affordable so?
